components/org.wso2.ppaas.manager.console/console/applications_form.jag

```diff
@@ -109,7 +109,7 @@ switch (formtype) {
 }
 
 if (dasStatsPublisherEnabled) {
-    meteringDashboardUrl = session.get('METERING_DASHBOARD_URL');
+    meteringDashboardUrl = application.get("meteringDashboardUrl");
```

Comment:
I'm getting the meteringDashboardUrl and monitoringDashboardUrl in same way (read them from cartridge-config.properties file at web app initialization and set them to application object in app.js.). Two urls are used in different views and the jag files they used to get this url values are different as below:
1. meteringDashboardUrl is needed in applications' topology view and topology view gets server side values from applications_form.jag when it's rendered.
2. monitoringDashboardUrl is needed in main menu view and slide-window menu view and these  pages get server side values from menu_generator.jag file when it's rendered.